test/tests/p5.Helpers.js

```diff
@@ -0,0 +1,35 @@
+const expect = chai.expect;
+
+describe('Testing helpers function', function () {
+  it('p5.prototype.freqToMidi helper function', function () {
+    const midi = p5.prototype.freqToMidi(880);
+    expect(midi).to.equal(81);
+  });
+
+  it('p5.prototype.midiToFreq helper function', function () {
+    const freq = p5.prototype.midiToFreq(720);
+    expect(freq).to.equal(9426054387186970000);
+  });
+
+  it('p5.prototype.noteToFreq helper function', function () {
+    const freq = p5.prototype.noteToFreq('C4');
+    expect(freq).to.equal(261.6255653005986);
+  });
+
+  it('p5.prototype.soundFormats helper function', function () {
+    // setting file format so that if we don't provide extension
+    // our file will be loaded because _checkFileFormats add it for us.
+
+    p5.prototype.soundFormats('mp3');
+    const file = p5.prototype._checkFileFormats('a');
+    expect(file).to.be.equal('a.mp3');
```

Comment:
This works, but ideally our test won't depend on a method with an underscore prefix (which, according to convention, is private and so could change or be deleted without us needing to update the API)

Reply:
can't think of a good way to do this off the top of my head...

Reply:
ok! I will try to come with some other function to test soundformat or I will leave it as it is.

Reply:
sorry! I am unable to find any better solution other than this.

Reply:
yeah, I'm not sure either but thanks for considering this feedback